modules/message.sh

```diff
@@ -231,6 +231,34 @@ function message {
             log_comment_and_exit1 "Error: You do not have an authorized access token" "To get access token, run $BASENAME message gitter apitoken"
           fi
           ;;
+        "")
+          if check_apitoken gitter; then
+            get_apitoken gitter
+            echo
+            echo "You have successfully configured your Gitter apitoken."
+            echo "Now you must provide a subcommand for $BASENAME message gitter listed below:"
+            echo
+            echo "  $BASENAME message gitter channels"
+            echo "     check for channels"
+            echo
+            echo "  $BASENAME message gitter send treehouses/Lobby \"Hi, you are very awesome\""
+            echo "     Sends a message to a gitter channel"
+            echo
+            echo "  $BASENAME message gitter show treehouses/Lobby"
+            echo "     Marks unread messages from a gitter channel to read"
+            echo
+            echo "  $BASENAME message gitter read treehouses/Lobby"
+            echo "     Receives and displays unread messages from a gitter channel"
+            echo
+            echo "  $BASENAME message gitter mark treehouses/Lobby"
+            echo "     Marks unread messages from a gitter channel to read"
+            echo
+          else
+            echo "You do not have a gitter apitoken setup."
+            echo "For instructions to configure your gitter apitoken"
+            echo "Please run: $BASENAME message gitter apitoken"
```

Comment:
This error message could also run `$BASENAME message gitter apitoken` at the end since that's the only decision path and lowers the drop-off rate.

Reply:
I will test that out, thanks.